roles/jenkins_builder/tasks/main.yml

```diff
@@ -37,6 +38,16 @@
     name: jenkins
     groups: wheel
 
+- import_tasks: setup_auth_rax.yml
+  when: location == 'rax'
```

Comment:
Why only for rax ?

Reply:
I want to use different keys for different groups of systems.